pkg/stub/imagestreams.go

```diff
@@ -175,7 +175,7 @@ func (h *Handler) upsertImageStream(imagestreamInOperatorImage, imagestreamInClu
 				// return the error so the caller can decide what to do
 				return err
 			}
-			if IsRetryableAPIError(err) || kerrors.IsConflict(err) {
+			if ShouldNotGoDegraded(err) {
 				logrus.Printf("CRDUPDATE: retryable error %s with imagestream update %s", err.Error(), imagestreamInCluster.Name)
```

Comment:
This is a pre-existing problem but this line causes panics, imagestreamInCluster is guaranteed to be nil (line 170 checks)

https://bugzilla.redhat.com/show_bug.cgi?id=2003683

Reply:
thanks for heads up @stbenjam  ... I've added a commit in this PR for now, but have also pinged @dperaza4dustbit about this 

we'll either address here with that commit , or he can do a separate PR, based on his taking ownership of samples (we are in the middle of the transition this week).